docs/user-guide/replicaset.md

```diff
@@ -0,0 +1,45 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Replica Set
+
+## What is a _replica set_?
+
+Replica set is the next generation replication controller. The only difference between a _replica set_ and a [_replication controller_](replication-controller.md) right now is the selector support. Replica set supports the new set-based selector requirements as described in the [labels user guide](labels.md#label-selectors) whereas a replication controller only supports equality-based selector requirements.
```

Comment:
@janetkuo 

> Provide an example of each selector here to demonstrate why it's more powerful?

It is in the example included below.

> We should link replication controller docs here

There is a link to RC doc already in the paragraph. The first occurrence of RC links to the doc.

> and notify the users that RS supersedes RC.

I am not very comfortable saying that yet. I will defer the final decision to @bgrant0607 though.